libafl_qemu/Cargo.toml

```diff
@@ -27,7 +28,7 @@ usermode = []
 clippy = [] # special feature for clippy, don't use in normal projects§
 
 [dependencies]
-libafl = { path = "../libafl", version = "0.8.1" }
+libafl = { path = "../libafl", version = "0.8.1", default-features = false, features = ["std", "derive", "llmp_compression", "rand_trait"] }
```

Comment:
I don't think we need rand_trait

Reply:
How do we find out if we don't need it for sure? I only removed `fork`, because I was not sure how to test properly if the other are also not needed. Just removing it and building any default fuzzer sounds a bit unsafe.

Reply:
Just try if compilation works, I'm 99% sure we don't need it

Reply:
It's not used by libafl_qemu, the fuzzers can still specify and use it directly

Reply:
Alright. `cargo build` works in `libafl_qemu` even without "rand_trait".